app/build.gradle

```diff
@@ -56,7 +56,7 @@ repositories {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile('com.twitter.sdk.android:twitter:1.3.0@aar') {
+    compile('com.twitter.sdk.android:twitter:1.4.0@aar') {
```

Comment:
it was the version suggested by the plugin at the time I started the update. Fixed.